examples/dense_bubble.py

```diff
@@ -76,12 +76,20 @@
     state.output.meanfields = {'rho':state.rhobar, 'theta':state.thetabar}
 
     # Set up advection schemes
-    Vtdg = FunctionSpace(mesh, "DG", 2)
+    ueqn = EulerPoincare(state, state.V[0])
+    rhoeqn = Advection(state, state.V[1], continuity=True)
+    supg = True
+    if supg:
+        thetaeqn = SUPGAdvection(state, state.V[2],
+                                 supg_params={"dg_direction":"horizontal"},
+                                 continuity=False)
+    else:
+        thetaeqn = EmbeddedDGAdvection(state, state.V[2],
+                                       continuity=False)
     advection_dict = {}
-    advection_dict["u"] = EulerPoincareForm(state, state.V[0])
-    advection_dict["rho"] = DGAdvection(state, state.V[1], continuity=True)
-    advection_dict["theta"] = SUPGAdvection(state, state.V[2], direction=[1])
-    # advection_dict["theta"] = EmbeddedDGAdvection(state, state.V[2], Vdg=Vtdg, continuity=False)
+    advection_dict["u"] = ThetaMethod(state, u0, ueqn)
```

Comment:
no, you can pass it in but it defaults to 0.5 because that's what I always use - would you rather it had to be explicitly set?